api/src/main/java/eu/coatrack/api/ProxyStates.java

```diff
@@ -0,0 +1,45 @@
+package eu.coatrack.api;
+
+/*-
+ * #%L
+ * coatrack-api
+ * %%
+ * Copyright (C) 2013 - 2021 Corizon | Institut für angewandte Systemtechnik Bremen GmbH (ATB)
+ * %%
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * 
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ * 
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * #L%
+ */
+
+public enum ProxyStates {
+
+    CRITICAL("Critical State", "critical-state"),
```

Comment:
i am wondering if "critical" is a good term. Maybe the gateway was switched off on purpose, maybe it should be down right now. 

Maybe we could call it "Not connected"?

Reply:
Ok, but what would be the meaning of "not connected" to the user?
We need to discuss this

Reply:
Whenever a critical is activated, the user would see a pop-up (Show me gateways status) and could choose just to ignore the state.
Add boolean flag to Proxy

Gateway Health Monitor
- Option to switch off notifications 
- Gateway Health status summary should only process/work for the ones that have the noticiation activated

How to activate/deactivate:
- We can use a slider or a button to switch off notifications

Reply:
Checkboxes could look nice and intuitive: 
<img width="290" alt="coatrack-gateway-health-monitor-example" src="https://user-images.githubusercontent.com/70953785/116666620-76050e80-a99b-11eb-88a8-2d5485b13d12.png">

Reply:
@ChristophBaierATB das ist okay, but we will not use checkboxes, we will be using switches.
And these switches will turn on the "grey" status

Reply:
The functionality is done.
The user can now select via the switch if the notifications should be shown or not